reset_local_db.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+export DB_URI=postgresql://myuser:mypassword@localhost:15432/simplelogin
+echo 'drop schema public cascade; create schema public;' | psql  $DB_URI 
```

Comment:
I wonder if we can use `dropdb simplelogin & createdb simplelogin` here from inside the container here?

Reply:
My local db users cannot create dbs. This way they don't need that permission while still being able to reset the db. 
What container do you mean?